addons/procurement/migrations/9.0.1.0/openupgrade_analysis.txt

```diff
@@ -1,10 +1,16 @@
 ---Fields in module 'procurement'---
-procurement  / procurement.order        / message_follower_ids (many2many): not a function anymore
-procurement  / procurement.order        / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
-procurement  / procurement.order        / message_follower_ids (many2many): type is now 'one2many' ('many2many')
-procurement  / procurement.order        / product_uos (many2one)        : DEL relation: product.uom
-procurement  / procurement.order        / product_uos_qty (float)       : DEL 
-procurement  / procurement.order        / website_message_ids (one2many): NEW relation: mail.message
+procurement  / procurement.order        / message_follower_ids (many2many)      : not a function anymore
```

Comment:
Two things about this:

* Don't change lines although the indentation is not so good for not increasing the diff.
* Why they are not included? Maybe we should improve the analysis method. Anyway, don't include here that extra lines, but it _work file.